packit_service/service/api/webhooks.py

```diff
@@ -120,3 +123,20 @@ def validate_signature():
             logger.warning(msg)
             logger.debug(f"X-Hub-Signature: {sig!r} != computed: {mac.hexdigest()}")
             raise ValidationFailed(msg)
+
+    @staticmethod
+    def interested():
+        """
+        Check X-GitHub-Event header for events we know we give a f...
+        ...finely prepared response to.
+        :return: False if we are not interested in this kind of event
+        """
+        uninteresting_events = {"integration_installation"}
```

Comment:
so what's the other event we're getting which we care about?

Reply:
`installation` (https://github.com/packit-service/packit-service/issues/42#issuecomment-552521935)